rasa/shared/nlu/training_data/training_data.py

```diff
@@ -60,6 +60,34 @@ def __init__(
 
         self._fill_response_phrases()
 
+    @staticmethod
+    def _load_lookup_table(lookup_table: Dict[Text, Any]) -> Dict[Text, Any]:
+        """Loads the actual lookup table from file if there is a file specified.
```

Comment:
should we add in the comment that this only applies to markdown and can be removed with 3.0?

Reply:
It applies not only to Markdown data but also to JSON data (and actually the existing test checks it with JSON)

Reply:
Let's add this to the docstring as this is helpful to kow.